recipes/cern-ldap

```diff
@@ -0,0 +1,4 @@
+(cern-ldap
+ :fetcher sourcehut
+ :repo "nbarrientos/cern-ldap.el"
+ :files ("cern-ldap.el"))
```

Comment:
(cern-ldap :fetcher sourcehut :repo "nbarrientos/cern-ldap.el")

Reply:
Thanks for the review. Sure, that's fixed. I've rewritten the history.